views/incubator/spindle.tpl

```diff
@@ -0,0 +1,195 @@
+  <li>Pen of chaos only produce black pens</li>
+  <li>
+    The pen is red if its price is below or equal to 1 CHF, else it is black
+  </li>
+  <li>
+    The pen is red if its length is below or equal to 12 cm, else it is black
+  </li>
+</ul>
+
+<p>
+  It is already a bit cumbersome to determinate theses relations, and that's
+  only for 4 products. Imagine having hundreds of producers, thousands of
+  product and many other features.
+  <br />
+  That's where using unsupervised machine learning helps: it discovers itself
+  the relations between each product, creating a "model" representing the data.
+  The output is not as readable as a human would do but takes way less time.
+  <br />
+  Usually, the model has a certain accuracy, as the whole data can rarely be
+  transformed in a single relation, so someone using it for prediction has to be
+  aware that it might return wrong results. It also only try to reproduce what
+  happened and can't predict the future, so if a producer wants to make cheap
+  black pens, it will misbehave.
+</p>
+
+<h2>Homomorphic Cryptography</h2>
+
+<p>
+  Cryptography is the science of encoding a message so that only the intended
+  recipient can decode it. It is used throughout computer science, when you
+  login into your computer, you connect to a website or even when you receive a
+  phone call.
+  <br />
+  To ensure that no malicious intermediary can decode the message, the sender
+  and the receiver have to share some kind of secret, such as a passphrase. As
+  such, even if someone intercepts the encoded message, they'll be unable to
+  decode it without this secret.
+</p>
+
+<p>
+  Let's create a very simple cryptography system which can only encode digits
+  from 0 to 9. We associate each digit to another digit in the same range, shift
+  it by a certain amount, in this case 4. This number of shift is actually the
+  secret that need to be shared with the receiver.
+</p>
+
+<table>
+  <tbody>
+    <tr style="border-bottom: thin solid">
+      <th scope="row">digit in clear</th>
+      <td>0</td>
+      <td>1</td>
+      <td>2</td>
+      <td>3</td>
+      <td>4</td>
+      <td>5</td>
+      <td>6</td>
+      <td>7</td>
+      <td>8</td>
+      <td>9</td>
+    </tr>
+    <tr>
+      <th scope="row">encoded digit</th>
+      <td>4</td>
+      <td>5</td>
+      <td>6</td>
+      <td>7</td>
+      <td>8</td>
+      <td>9</td>
+      <td>0</td>
+      <td>1</td>
+      <td>2</td>
+      <td>3</td>
+    </tr>
+  </tbody>
+</table>
+
+<p>
+  For example, if you want to send 7 to someone, you'll first encode it as 1 and
+  send it to the receiver, who will decode it to 7. So we are able to send a
+  message, so that only the person having the secret can decode it.
+</p>
+
+<p>
+  Homomorphic encryption is a special kind of encryption where you can take if
+  you apply a mathematical operation between two encoded message, it will be the
+  same as doing this operation on decoded message.
```

Comment:
This sentence does not make sense to me...

Reply:
Oups, I started with a sentence idea and finished with another one :sweat_smile: Is the following making more sense?

>   Homomorphic encryption is a special kind of encryption where when you apply a mathematical operation between two encoded message, it will be the same as doing this operation on the decoded messages. Let's say you want to find out the sum of some encoded message, you can add theses messages and decode the resulting one to obtain the sum, as if the messages weren't encoded in the first place.

Reply:
Excellent :)